var/spack/repos/builtin/packages/py-patsy/package.py

```diff
@@ -11,6 +11,7 @@ class PyPatsy(PythonPackage):
     homepage = "https://github.com/pydata/patsy"
     pypi = "patsy/patsy-0.4.1.zip"
 
+    version('0.5.2', sha256='5053de7804676aba62783dbb0f23a2b3d74e35e5bfa238b88b7cbf148a38b69d')
     version('0.5.1', sha256='f115cec4201e1465cd58b9866b0b0e7b941caafec129869057405bfe5b5e3991',
             url="https://pypi.io/packages/source/p/patsy/patsy-0.5.1.tar.gz")
     version('0.4.1', sha256='dc1cc280045b0e6e50c04706fd1e26d2a00ea400aa112f88e8142f88b0b7d3d4')
```

Comment:
Need minimum dep version for numpy too

Reply:
I added 
    depends_on('py-numpy',       type=('build', 'run'))
    depends_on('py-numpy@1.4:',  type=('build', 'run'), when='@0.5.1:')